src/state_control.cpp

```diff
@@ -39,7 +39,7 @@ using namespace std;
 #define RESET "\e[0m"
 
 // State machine
-enum controller_state
+enum tracker_state
```

Comment:
It doesn't need to. I just did it for consistency. It seems like you are calling controllers trackers now.